skins/cat17/address-form/src/types.ts

```diff
@@ -28,7 +28,7 @@ export interface Helper {
 
 export interface InputField {
 	name: string,
-	value: string,
+	value: string | boolean,
```

Comment:
I'm not against this but I'd like to mention that we had the same challenge with the `newsletterOptIn` for the address form in `laika` and there instead of doing `string | boolean` we extracted the boolean property out of that interface, because that's technically incorrect as all other fields are not boolean but now we allow them to be. 
```js
addressType: AddressTypeModel.PERSON,
newsletterOptIn: false,
values: {
    salutation: '',
    title: '',
    ...
}, 
```
```js
export interface InputField {
    name: string,
    value: string,
    pattern: string,
    optionalField: boolean
}

export interface FormData {
    [key: string]: InputField
}

export interface AddressState {
    isValidating: boolean,
    addressType: AddressTypeModel,
    newsletterOptIn: boolean,
    ...
}
```

Also if we do that here and then port it to `laika` we might have to do some additional adjustment to the store there etc.

Reply:
> technically incorrect as all other fields are not boolean but now we allow them to be

Thanks for pointing out! That's a potential source of errors and headaches and I will change it.